server/src/modules/data/usecases/getDemandesRestitutionIntentions/deps/getFilters.query.ts

```diff
@@ -1,4 +1,5 @@
 import { ExpressionBuilder, sql } from "kysely";
+import _ from "lodash";
```

Comment:
Histoire d'alléger tout ça

```suggestion
import capitalize from "lodash/capitalize";
```